docker-compose.tbls.yml

```
@@ -0,0 +1,36 @@
+services:
+  db-temp:
+    image: mariadb:11.8
+    restart: no
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)